specification/personal-demographics.yaml

```diff
@@ -464,7 +464,7 @@ paths:
         ### Restricted patients
         Some patients are tagged as [restricted](https://digital.nhs.uk/services/demographics/restricting-access-to-a-patients-demographic-record) and are sometimes known as sensitive patients. When performing a search, restricted patients can be returned; however, location sensitive fields such as `address`, `telecom`, `contact` and `generalPractitioner` are removed.
 
-        If `address-postcode` or `general-practitioner` are included in the search parameters, no restricted patients are returned even if a match is identified.
+        If `address-postalcode` or `general-practitioner` are included in the search parameters, no restricted patients are returned even if a match is identified.
```

Comment:
Should this be "If `address-postalcode`, `address-postcode`, or `general-practitioner`..."

At least whilst we support both postcode properties